dask_geomodeling/geometry/aggregate.py

```diff
@@ -176,7 +176,7 @@ def __init__(
         max_pixels=None,
         column_name="agg",
         auto_pixel_size=False,
-        *args,
+        *args
```

Comment:
Je gebruikt geen "black"? Die voegt de trailing comma's juist toe.

Reply:
`*args, ` is invalid syntax in python 3.5 . Black heeft logica die een file bekijkt en raadt of python 3.5 compatibility nodig is..  Dus hij voegt m niet toe hier.